cmd/wave/main.go

```diff
@@ -244,9 +245,24 @@ func main() {
 
 	conf.Keychain = kc
 
-	if auth.ClientID != "" && auth.ClientSecret != "" && auth.ProviderURL != "" && auth.RedirectURL != "" {
+	if conf.IDE {
+		conf.Proxy = true // IDE won't function without proxy
+	}
```

Comment:
No need for this extra check, already at line 267.

Reply:
Deleted